client/screens/AssetDetailsScreen.tsx

```
@@ -211,18 +242,11 @@ export default function AssetDetailsScreen() {
-    if (asset?.assetNumber) {
-      return asset.assetNumber;
-    }
-
-    const manufacturerModel = [asset?.manufacturer, asset?.model]
-      .filter(Boolean)
-      .join(" ")
-      .trim();
-
-    return manufacturerModel || "Asset Details";
-  }, [asset?.assetNumber, asset?.manufacturer, asset?.model]);
+  const assetDisplayTitle = useMemo(() => buildAssetDisplayTitle(asset), [asset]);
```

ESLint

[error] Replace `()·=>·buildAssetDisplayTitle(asset),·[asset]` with `⏎····()·=>·buildAssetDisplayTitle(asset),⏎····[asset],⏎··`

(prettier/prettier)